frontend/src/lib/components/PropertyFilters/PropertyFilter.js

```diff
@@ -58,11 +58,11 @@ function PropertyPaneContents({
                                           propkey,
                                   }
                         }
-                        onChange={(type, value) =>
+                        onChange={(type, _value) =>
```

Comment:
FYI @samwinslow fixed this shadow variable issue (pre-existing), but think it demonstrates our typical bias for action / tread on toes workflow.